tools/macos/README.md

```diff
@@ -7,7 +7,7 @@ One way to develop or deploy OpenWhisk on a Mac is to use [docker-machine](https
 The following are required to build and deploy OpenWhisk from a Mac host:
 
 - [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-- [Docker 1.9.1](https://docs.docker.com/engine/installation/mac/) (including `docker-machine`)
+- [Docker 1.12.0](https://docs.docker.com/engine/installation/mac/) (including `docker-machine`)
```

Comment:
I want one consistent docker version on all environments. There is no docker bosh release for 1.12.1 yet. Since we are planning on moving away from bosh this may become a non-issue soon.